lib/transcoder.ex

```diff
@@ -1,9 +1,45 @@
 defmodule Transcoder do
```

Comment:
`@moduledoc` with a short description is always helpful. Also, non-obvious public functions deserve a doc as well.

Reply:
Nit:
I'm not sure if we want this to be in the root of the app...? On one hand, our app is a transcoder, but maybe it'll eventually become somethign more so we might want to leave some flexibility?

Reply:
Moved to `Transcoder.Server`